snippets/markdown.json

```diff
@@ -1,4 +1,27 @@
 {
+    "YAML Header": {
+        "prefix": "---",
+        "body": [
+            "---",
+            "title: ${1:title}",
+            "author: ${2: author_name}",
+            "date: ${3:\"`r Sys.Date()`\"}",
+            "output: ${5:pdf_document}",
+            "---"
+        ]
+    },
+    "Insert Inline Chunck": {
+        "prefix": "`",
+        "body": "`${0}`"
+    },
```

Comment:
We should remove this snippet as it's going to clash with the one below it. Also, if users have an auto-pairing plugin/configuration then it renders this moot.